File: jdt-dpgen-cli/src/main/resources/dpgen-output/src/main/java/com/gof/behavioral/visitor/ConcreteVisitor2.java
package com.gof.behavioral.visitor;
public class ConcreteVisitor2 implements Visitor {

	@Override
	public void visitConcreteElementA(ConcreteElementA concreteElementA) {
		concreteElementA.operationElementA();
	}

	@Override
	public void visitConcreteElementB(ConcreteElementB concreteElementB) {
		concreteElementB.operationElementB();
	}
}

File: jdt-dpgen-cli/src/main/resources/dpgen-output/src/main/scala/com/gof/behavioral/visitor/ConcreteVisitor2.scala
package com.gof.behavioral.visitor

//remove if not needed
import scala.collection.JavaConversions._

class ConcreteVisitor2 extends Visitor {

  override def visitConcreteElementA(concreteElementA: ConcreteElementA) {
    concreteElementA.operationElementA()
  }

  override def visitConcreteElementB(concreteElementB: ConcreteElementB) {
    concreteElementB.operationElementB()
  }
}
